src/pages/ShopingCart.page.js

```diff
@@ -22,4 +22,29 @@ export class ShopingCartPage extends BaseSwagLabPage {
     async removeCartItemById(id) {
         await this.cartItems.nth(id).locator(this.removeItemSelector).click();
     }
+
+    async productInfoByID(id) {
```

Comment:
Why do we have same method as we had? Lets find a way to reuse it!

Reply:
Updated